openapi/smartevents_mgmt_v2.yaml

```diff
@@ -281,6 +284,289 @@ paths:
                 $ref: '#/components/schemas/ErrorsList'
       security:
       - bearer: []
+  /api/smartevents_mgmt/v2/bridges/{bridgeId}/processors:
+    get:
+      tags:
+      - Processors
+      summary: Get the list of Processors of a Bridge instance
+      description: Get the list of Processors of a Bridge instance for the authenticated
+        user.
+      operationId: ProcessorsAPI_listProcessors
```

Comment:
Can we keep the same verbs of the bridge API? like `ProcessorAPI_getProcessors`. 
They end up as names of the sdk client methods.